analyzers/Threatcrowd/threatcrowd_analyzer.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+import requests
+from cortexutils.analyzer import Analyzer
+
+class Threatcrowd(Analyzer):
+
+    URI = "https://www.threatcrowd.org/searchApi/v2"
+
+    def summary(self, raw):
+        taxonomies = []
+
+        level = "info"
+        value = "None"
+
+        if 'votes' in raw:
+            r = raw.get('votes')
+            value = r
+            if r == 1:
+                level = "safe"
+            elif r == 0:
+                level = "suspicious"
+            elif r == -1:
+                level = "malicious"
+
+        taxonomies.append(self.build_taxonomy(level, "Threatcrowd", "votes", value))
+
+        result = {"taxonomies": taxonomies}
+        return result
+
+    def run(self):
+        Analyzer.run(self)
+
+        if (self.data_type == 'domain' or self.data_type == 'ip' or self.data_type == 'email'):
+            try:
+                response = requests.get("{}/{}/report/".format(self.URI, self.data_type), {self.data_type: self.get_data()})
```

Comment:
As above: default e-mail datatype is `mail`.

Something like `self.data_type if self.data_type != 'mail' else 'email'` would be possible. Requests to threat crowd contains then `email` as a datatype.